content/docs/command-reference/remote/modify.md

```diff
@@ -827,19 +827,30 @@ $ export OSS_ENDPOINT='endpoint'
   $ dvc remote modify --local myremote keyfile /path/to/keyfile
   ```
 
-- `password` - a private key passphrase or a password to access the remote.
+- `password` - a password to access the remote.
 
   ```dvc
   $ dvc remote modify --local myremote password mypassword
   ```
 
-- `ask_password` - ask for a private key passphrase or a password to access the
-  remote.
+- `ask_password` - ask for a password to access the remote.
```

Comment:
Q: is it true or false by default? (same for the `ask_passphrase`) . There was a recent complaint on Discord for `dvc exp run` hanging on trying to pull things from an SSH remote (which is by itself is not if it can be handled better) - is this change related to that?

Reply:
>  is it true or false by default? 

It's False (not enabled) by default

> There was a recent complaint on Discord for dvc exp run hanging on trying to pull things from an SSH remote (which is by itself is not if it can be handled better) - is this change related to that?

Sounds unrelated. 
This change is about https://github.com/iterative/dvc/issues/8137#issuecomment-1218341188